python/setup.py

```diff
@@ -0,0 +1,34 @@
+"""
+File: setup.py
+
+Description:
+    Sample setup script.
+
+Copyright Provizio Ltd., 2022
+Author: Dane Mitrev
+"""
+import os
+from setuptools import setup
+
+
+def read(file_name):
+    return open(os.path.join(os.path.dirname(__file__), file_name)).read()
+
+
+setup(
+    name="src",
+    version="0.0.1",
+    author="Provizio Developer",
+    author_email="namesurname@provizio.ai",
```

Comment:
Here, you surely meant @proviz.io instead @provizio.ai

Reply:
Actually the website works with both, maybe not the email tho, need to check. Anyways it is a placeholder, I will update it. Thanks!